db/agents.json

```diff
@@ -1610,13 +1712,18 @@
     },
     {
       "regex": "^(AppleCoreMedia.+Apple Watch|atc.+watchOS|\\(null\\).+watchOS)",
```

Comment:
this is more expansive than the IAB guidelines
https://iabtechlab.com/blog/apple-watch-os-podcast-filtering-guidance/
```
User Agents:
That begin with atc/ and include watchOS (for example atc/1.0 watchOS)
Or contain (null)/(null) watchOS*
```
Looks like you added filtering out `AppleCoreMedia.+Apple Watch` too

Reply:
I mean, I agree those are Apple Watch, for identifying
Not sure they should be filtered out - that might be the "legit" traffic from them.

Reply:
Good point - will separate those out.

Though FWIW - they're actually very little of our Apple watch traffic.  Counts for November:

```
114602332 -> all downloads
29272     -> remote_agent like 'AppleCoreMedia%Apple Watch%'
7100376   -> remote_agent like 'atc%watchOS%'
16214     -> remote_agent like '(null)%watchOS%'
```

Reply:
cool cool cool